src/components/Messages.vue

```diff
@@ -97,9 +97,8 @@ export default {
       const lastSeenMsgId = this.messages[0] ? this.messages[0].entry.uuid : null
       await this.listMessagesPage({
         channel: this.channel,
-        earlier_than: this.earliestDate - (60 * 60 * 1000 * 1000), // TODO: this is a hack to work around a dna bug. This should be removed once that bug is fixed
+        earlier_than: this.earliestDate,
```

Comment:
This should be `earliest_seen` right? And the DNA is programmed to handle null as current time so I think earliestDate should be change to be nullable

Reply:
If `now` is passed for `earliest_seen`, it will exclude any messages from the current hour, that's why it's important to pass null instead

Reply:
I don't think so. The place where we want the current time is in the other call, that I pointed below. 

And this is where you point it to the earliest_seen which I am assuming is `this.earliestDate`.

But I'll defer to @robbiecarlton for this change

Reply:
Oh I think you're right

Reply:
but the zome call arg should still be renamed to `earliest_seen` right?

Reply:
Yes @robbiecarlton